frontend/src/services/authServices.ts

```diff
@@ -1,19 +1,23 @@
-import axios from "axios"
+import axios from 'axios'
 
-const API_URL = ""
+const API_URL = 'http://10.10.0.120:8080/users/'
 
+const register = (data: any) => {
+  console.log(data)
+  return axios.post(`${API_URL}register`, data)
+}
+
+const login = (data: any) => {
+  return axios.post(`${API_URL}login`, data)
+}
+
+const logout = (setUser: any) => {
```

Comment:
No backend call? Will the token still be valid on the BE?

Reply:
Backend did not finished that yet.